modules/drivers/mongo/test/metabase/driver/mongo_test.clj

```diff
@@ -197,36 +196,44 @@
              {:schema nil, :name "reviews"}}
             (:tables (driver/describe-database :mongo (mt/db)))))))
 
-(deftest ^:parallel describe-table-test
+(defmacro with-describe-table-query-depths
```

Comment:
```suggestion
(defmacro with-describe-table-query-depths!
```

To make it clear this is thread-unsafe and that it's not ok for use in parallel tests